src/connection.ts

```diff
@@ -935,6 +938,15 @@ class Connection extends EventEmitter {
         this.config.options.useUTC = config.options.useUTC;
       }
 
+      if (config.options.validateBulkLoadParameters !== undefined) {
+        if (typeof config.options.validateBulkLoadParameters !== 'boolean') {
+          throw new TypeError('The "config.options.validateBulkLoadParameters" property must be of type boolean.');
+        }
+
+        deprecate('This "validateBulkLoadParameters" property will be depercated in the next major version of `tedious`.');
```

Comment:
I am not sure that I suppose to add the message here. I thought that users may want to know that this property is going to be deprecated when they are actually using it which means that when "config.options.validateBulkLoadParameters !== undefined".  However, I am not sure whether I am correct about this.